app/Theme.php

```diff
@@ -118,23 +118,22 @@ public function register_assets() {
 	 * Register post types
 	 */
 	public function register_post_types() {
-		new \Boilerplate\Theme\Post_Type\Hero();
-		new \Boilerplate\Theme\Post_Type\Example();
+		new \Boilerplate\Theme\Post_Type\Employee();
 	}
 
 	/**
 	 * Register post types
 	 */
 	public function register_taxonomies() {
-		new \Boilerplate\Theme\Taxonomy\Example();
+		new \Boilerplate\Theme\Taxonomy\Department();
 	}
```

Comment:
you will need to upgrade call of post types and taxonomies with new theme framework 2.1. see develop branch